docs/APIs/storage/block_device.md

```diff
@@ -28,6 +28,14 @@ Block devices indicate their block sizes through the `get_read_size`, `get_progr
 
 As a rule of thumb, you can use the erase size for applications that use a single block size (for example, the FAT file system).
 
+## Utility block devices
+
+mbed OS contains several utility block devices to give you better control over how storage is allocated.
+
+- The [slicing block device](https://github.com/ARMmbed/mbed-os/blob/master/features/filesystem/bd/SlicingBlockDevice.h) allows you to partition storage into smaller block devices that you can use independentally.
```

Comment:
In general, we try to avoid "allows". What do you think about changing this bullets to "With the slicing block, you can partition storage..."? Does that still keep the meaning the same?

Please apply this change to the following two bullets, too.